ext/mixed/js/display.js

```diff
@@ -46,7 +46,14 @@ class Display {
         this.index = 0;
         this.audioPlaying = null;
         this.audioFallback = null;
-        this.audioSystem = new AudioSystem({getAudioUri: this._getAudioUri.bind(this)});
+        this.audioSystem = new AudioSystem({
+            audioUriBuilder: {
+                async getUri(definition, source, details) {
+                    return await apiAudioGetUri(definition, source, details);
+                }
+            },
```

Comment:
I think this would be easier to read as
```javascript
            audioUriBuilder: {
                getUri: async (definition, source, details) => {
                    return await apiAudioGetUri(definition, source, details);
                }
            },
```

Reply:
Curse you Javascript for giving us multiple ways to do the same thing!

0a9f5f72973880dd29b54f71ad727ad3ac42d7aa